cartography/data/jobs/analysis/keycloak_inheritance.json

```diff
@@ -0,0 +1,30 @@
+{
+  "statements": [
```

Comment:
I don't think I see a cleanup step for this, might want to add that. Other analysis jobs have a few steps where they remove stale attributes or relationships before computing new ones